delivery_carrier_label_ups/models/stock_picking.py

```diff
@@ -170,6 +170,14 @@ def _ups_shipping_data(self):
             )
         )
 
+    def _ups_return_service(self):
+        """Return a dict describing the UPS Return Label code"""
+        self.ensure_one()
+        return dict(
+            # UPS Electronic Return Label (ERL)
+            Code='8',
```

Comment:
The return service type was implemented by our team for our customer. If we can't get it to work for us, it will not work for anybody and it has to be reverted as a whole.

Reply:
If we need it anyway for our project, do you see a chance to implement it here rather than in our customizations?

Reply:
Yes, I will try to implement it here. I added https://github.com/OCA/delivery-carrier/pull/418/commits/9c5e14e8acce70e9e3ff2c3d37cf4bd3c5b96748 which is still a work in progress as I don't have yet a full understanding of the UPS specifications.

Reply:
I put this PR to ready for review. I will update it in case something will come out from the tests at the customer's side.

Reply:
weird thing is this worked in the test environment of UPS...

Reply:
@hbrunn I don' know why. Did they maybe change something in UPS api? Or is it maybe a matter of configuration on UPS customer side?

Reply:
thankfully I've forgotten most of the super unwieldy hard to search non-indexed PDFs that UPS calls the documentation for their API, so I happily leave this to you :-)